### DISCUSSION TITLE
Adjust shm size HASS Add-on

### DISCUSSION THREAD

**Discussion by Mustang1966** [blakeblackshear/frigate#485] [Ask A Question] [answered]
Hello,

How can i adjust the shm size in the Hass Addon? I have 6 camera's, but getting a **bus error** when i add more the 3 camera's.

Using 1.2 beta

**Comment by blakeblackshear** [comment]
It's not possible

**Comment by blakeblackshear** [comment, marked as answer]
The shm size cannot be set per container for HomeAssistant Addons. You must set `default-shm-size` in `/etc/docker/daemon.json` to increase the default shm size. This will increase the shm size for all of your docker containers. This may or may not cause issues with your setup. https://docs.docker.com/engine/reference/commandline/dockerd/#daemon-configuration-file
